lang/python27/Portfile

```diff
@@ -124,6 +124,15 @@ post-destroot {
     ln -s ${framewdir}/share/man/man1/python${branch}.1 ${destroot}${prefix}/share/man/man1/
     ln -s ${framewdir}/Python ${destroot}${prefix}/lib/libpython${branch}.dylib
 
+    # Without this, LINKFORSHARED is set to
+    # ... $(PYTHONFRAMEWORKDIR)/Versions/$(VERSION)/$(PYTHONFRAMEWORK)
+    # (this becomes Python.framework/Versions/2.7/Python) which doesn't
+    # quite work (see ticket #15099); instead specifically list the
+    # full path to the proper Python framework file (which becomes
+    # ${prefix}/Library/Frameworks/Python.framework/Versions/2.7/Python)
+    reinplace {s|^\(LINKFORSHARED=.*\)$(PYTHONFRAMEWORKDIR).*$|\1 $(PYTHONFRAMEWORKINSTALLDIR)/Versions/$(VERSION)/$(PYTHONFRAMEWORK)|} \
```

Comment:
The value is in the Makefile as ABIFLAGS, in 3.5 at least. https://www.python.org/dev/peps/pep-3149/